galleries/examples/text_labels_and_annotations/label_subplots.py

```diff
@@ -3,54 +3,57 @@
 Labelling subplots
 ==================
 
-Labelling subplots is relatively straightforward, and varies,
-so Matplotlib does not have a general method for doing this.
+Labelling subplots is relatively straightforward, and varies, so Matplotlib
+does not have a general method for doing this.
 
-Simplest is putting the label inside the axes.  Note, here
-we use `.pyplot.subplot_mosaic`, and use the subplot labels
-as keys for the subplots, which is a nice convenience.  However,
-the same method works with `.pyplot.subplots` or keys that are
-different than what you want to label the subplot with.
+The examples here rely on the ability of `~.Axes.annotate` to conveniently
+position text at a given physical offset (in fontsize units or in points) away
+from a corner of the Axes.
+
+Note, here we use `.pyplot.subplot_mosaic`, and use the subplot labels as keys
+for the subplots, which is a nice convenience.  However, the same method works
+with `.pyplot.subplots` or keys that are different than what you want to label
+the subplot with.
 """
 
 import matplotlib.pyplot as plt
 
-import matplotlib.transforms as mtransforms
-
+# %%
 fig, axs = plt.subplot_mosaic([['a)', 'c)'], ['b)', 'c)'], ['d)', 'd)']],
                               layout='constrained')
-
 for label, ax in axs.items():
-    # label physical distance in and down:
-    trans = mtransforms.ScaledTranslation(10/72, -5/72, fig.dpi_scale_trans)
-    ax.text(0.0, 1.0, label, transform=ax.transAxes + trans,
-            fontsize='medium', verticalalignment='top', fontfamily='serif',
-            bbox=dict(facecolor='0.7', edgecolor='none', pad=3.0))
-
-plt.show()
+    # Put the label
+    # - at the top left corner (axes fraction (0, 1)),
+    # - offset half-a-fontsize right and half-a-fontsize down
+    #   (offset fontsize (+0.5, -0.5)),
+    # i.e. just inside the axes.
```

Comment:
I'm not sure if this would be better in the text above instead of a comment, but unfortunately, the CI-built docs have been garbage-collected now.

Reply:
Re-triggered a CI run.